### PR TITLE
fix(server/main): sendDoctorAlert

### PR DESCRIPTION
**Pull request by mockdot**
## Description

- send doctor alert
- correct notification message

## Checklist

- [x] I have personally loaded this code into an updated Qbox project and checked all of its functionality.
- [x] My code fits the style guidelines.
- [x] My PR fits the contribution guidelines.

### CODE REVIEW

File: server/main.lua [hunk 1]
```diff
@@ -143,10 +143,11 @@ end)
 local function sendDoctorAlert()
 	if doctorCalled then return end
 	doctorCalled = true
-	local _, doctors = QBCore.Functions.GetDutyCountJob('ambulance')
-	for i = 1, #doctors do
-		local doctor = doctors[i]
-		TriggerClientEvent('ox_lib:notify', doctor.PlayerData.source, { description = Lang:t('info.dr_needed'), type = 'inform' })
```
Comment (Manason) [inline server/main.lua]: this doesn't work?

Comment (mockdot) [reply server/main.lua]: It didn't and I did not check the function, I see that it should return players as the second value which we are getting in pairs here, I'll check further, might just be the way that playerdata.source is pulling the number, but I was getting that doctor. was a number not a table.

Comment (mockdot) [reply server/main.lua]: corrected

File: server/main.lua [hunk 2]
```diff
@@ -143,10 +143,10 @@ end)
 local function sendDoctorAlert()
 	if doctorCalled then return end
 	doctorCalled = true
-	local _, doctors = QBCore.Functions.GetDutyCountJob('ambulance')
+	local _, doctors = QBCore.Functions.GetDutyCountType('ems')
 	for i = 1, #doctors do
 		local doctor = doctors[i]
```
Comment (ChatDisabled) [inline server/main.lua:148]: @Manason why is this localized instead of directly pointing at it in the next line?

Comment (Manason) [reply server/main.lua:148]: doctors is used twice

**Pull request merged**